src/core/datatypes/trees/Clade.cpp

```diff
@@ -519,6 +501,26 @@ bool Clade::overlaps(const Clade& c) const
     return false;
 }
 
+set<Taxon> Clade::intersection(const Clade& c) const
```

Comment:
Could you write a one-line comment above what this method is doing and should be used for?

Reply:
Done.